content/docs/1.17/operator.md

```diff
@@ -19,29 +19,64 @@ While multiple operators might coexist watching the same set of namespaces, whic
 The Jaeger Operator version tracks one version of the Jaeger components (Query, Collector, Agent). When a new version of the Jaeger components is released, a new version of the operator will be released that understands how running instances of the previous version can be upgraded to the new version.
 {{< /info >}}
 
+## Install modes
+
+The Jaeger Operator can be installed to watch for new Jaeger resources either in the whole cluster or in specific namespaces.
+
+When watching all namespaces, use the `cluster_role.yaml` and `cluster_role_binding.yaml` files while installing the operator, and set the `WATCH_NAMESPACE` to empty in the `operator.yaml` file. With this in place, the Jaeger Operator will:
```

Comment:
We're talking about what YAML files to use before we've even told the user that they have to install YAML files, which comes in the next section.  I think what we might want to do here is say something like "To install in Mode A, you must install these YAML files [then list].  To install in Mode B, you must install these YAML files [then list for that mode].

Reply:
I removed the mention to those files, keeping this intro generic. A list of files are still the same place as it was before, but let me know if they should be clearly split in two sections.